tests/Feature/AccessTokenTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+
+namespace Tests\Feature;
+
+
+use App\Passport\Client;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Illuminate\Support\Str;
+use Tests\TestCase;
+
+class AccessTokenTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    private $client;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->client = factory(Client::class)->create([
+            'redirect' => 'http://example.org/callback',
+            'personal_access_client' => false,
+            'password_client' => false,
+        ]);
+    }
+
+    /** @test */
```

Comment:
From a PHPUnit perspective, dont need the`@test` annotation if the test names start with `test`